server/src/com/cloud/user/AccountManagerImpl.java

```diff
@@ -2196,6 +2193,11 @@ private UserAccount getUserAccount(String username, String password, Long domain
                 s_logger.debug("Unable to authenticate user with username " + username + " in domain " + domainId);
             }
 
+            if (userAccount == null) {
+                s_logger.warn("Unable to find an user with username " + username + " in domain " + domainId);
```

Comment:
This may also happen if ldap user exists but authn. failed due to incorrect password. The message wouldn't be correct. Is it intentional?